neuro_san/internals/run_context/langchain/llms/default_llm_factory.py

```diff
@@ -361,7 +301,7 @@ def create_llm_resources_with_client(self, config: Dict[str, Any],
         found_exception: Exception = None
         for llm_factory in self.llm_factories:
             try:
-                llm_resources = llm_factory.create_llm_resources_with_client(config, llm_client)
+                llm_resources = llm_factory.create_llm_resources(config)
```

Comment:
Prefer the create_llm_resources() method before trying the older create_chat_model() method.